Add null validation on attrib_type_id column on AttribDefaultValue table

danidoni opened the pull request:
We have a database backed null constraint on that column but no validation in the model.

saraycp commented:
@danidoni I added a third commit to solve the issue detected by the minitest tests `test_create_type` and `test_ceate_type_via_group`.

danidoni commented:
> @danidoni I added a third commit to solve the issue detected by the minitest tests `test_create_type` and `test_ceate_type_via_group`. It just needs some review.

Alright, I was trying to rework the association so when doing the `attrib_type.default_values << Attrib

> @danidoni I added a third commit to solve the issue detected by the minitest tests `test_create_type` and `test_ceate_type_via_group`. It just needs some review.

Thanks!
